build.sh

```diff
@@ -5,80 +5,109 @@
 # 2022-07-31: adding extra rc.local, setting rc build script
```

Comment:
In case build.sh is **not** executable, please change to 0755 (didn't check)

Reply:
this is fixed. I had it set to non-executable because I'm copying to build-rc.sh and running that with some changes to image name. This is non-issue now as default is build without upload.